### DISCUSSION TITLE
[Question] Ten-element alloy structural stability Bulk simulation shows phase transition issue with DeepMD + LAMMPS

### DISCUSSION THREAD

**Discussion by K-L-Wu** [deepmodeling/deepmd-kit#4895] [Q&A]
Dear DeePMD developers and team,
I would like to express my sincere appreciation for your outstanding work in developing such an accurate and reliable potential model. Your contributions have significantly advanced our ability to study materials at the atomic scale and have enabled substantial progress in our research.

I am currently using a Deep learning potential (DeepMD) trained on 10 elements alloy to simulate material structure stability in LAMMPS. The aim is to study structural stability for different compositions.

Background:  
- Successfully trained a Deep Learning Potential (DLP). 
  - For detailed information regarding DLP training, please kindly refer to Table 1,2,3. 
  (Tables 1: Descriptor Settings, Table 2: Loss Function Weights, and Table 3: Summary table of DFT-calculated structures for Deep Learning potential training dataset)
- Structural stability simulations were conducted using LAMMPS.
  - Simulation protocol: Structure minimization → NPT ensemble simulation

Observed Issue:
- Under identical simulation conditions, I observe two distinct behaviors:
  - Non-equimolar composition
    - Minimize stage: remains FCC 
    - NPT stage: transforms from FCC → Other (Figure 1) 
  - Equimolar composition
    - Minimize stage: transforms from FCC → BCC (Figure 2)
    - NPT stage: remains BCC 

Questions: 
1. Could this be due to insufficient bulk structure data in the DeepMD training dataset, resulting in inaccurate phase description?
2. Is it necessary to adjust the training parameters?
3. Is it necessary to include some 5-element or 6-element alloys (fewer than 10 elements) in the training dataset?

Thank you for your time and suggestions. I look forward to your responses.

### Table 1：Descriptor Settings

| Descriptor | Parameter |
|------------|-----------|
| Descriptor | se_e2_a   |
| type       | se_e2_a   |
| rcut       | 9.18000000000001 |
| rcut_smth  | 5.2200000001 |

### Table 2：Loss Function Weights

| Weight | Value |
|--------|-------|
| start_pref_f | 1000 |
| limit_pref_f | 5    |
| start_pref_v | 10   |
| limit_pref_v | 0.01 |
| start_pref_e | 0.01 |
| limit_pref_e | 1    |

### Table 3：Summary table of DFT-calculated structures for Deep Learning potential training dataset

| Structure Source   | Material Elemental Composition                 | Crystal Type         | Simulation Conditions               | Simulation Temperature | Quantity |
|:-----------------:|:---------------------------------------------:|:------------------:|:---------------------------------:|:-------------------:|:-------:|
| Materials Project  | Unary<br>Binary<br>Ternary<br>Quaternary     | FCC、BCC、HCP       | Bulk vc-md                         | 300K                 | 447     |
| Self-generated     | Denary                                       | FCC、BCC、HCP       | Bulk vc-md                         | 300K                 | 48      |
| Self-generated     | Denary                                       | FCC、BCC、HCP       | Surface(100)、(111) vc-md          | 300K                 | 52      |

Figure 1：Non-equimolar composition | NPT stage transforms from FCC → Other

<img width="1782" height="821" alt="image" src="https://github.com/user-attachments/assets/3ddda280-c7c6-4b4e-82f9-e95c3bd88eaf" />

Figure 2：Equimolar composition | Minimize stage

<img width="1758" height="826" alt="image" src="https://github.com/user-attachments/assets/ead9f136-26a2-430a-8107-28b7e1f59053" />

**Comment by wanghan-iapcm** [comment]
Firstly I would like to recommend using the "dpa1" descriptor for higher generalization, as discussed in https://www.nature.com/articles/s41524-024-01278-7, which has shown that the generalizability of dpa1 in high-entrpy alloy systems is much higher than standard dp descriptor.
- you may use "dpa1" descriptor with "attn_layer": 2 for higher accuracy or
- "attn_layer": 0 for better efficiency. 

regarding your questions:
1.  Could this be due to insufficient bulk structure data in the DeepMD training dataset, resulting in inaccurate phase description?
I am not sure about the method you used to generate training data, but the data coverage does matter, please see the discussion in https://github.com/deepmodeling/deepmd-kit/discussions/4873

2.  Is it necessary to adjust the training parameters?
please see my suggestion on descriptor.

3.   Is it necessary to include some 5-element or 6-element alloys (fewer than 10 elements) in the training dataset?
If your target is 10-element systems, then it is not necessary.

**Reply by K-L-Wu** [reply]
Thank you for your response, Dr. Wang Han. I will continue my work by following your suggestions.
